storage/file.go

```diff
@@ -51,6 +51,24 @@ type FileStream struct {
 	ContentMD5 string
 }
 
+// FileRequestOptions will be passed to misc file operations.
+// Currently just Timeout but will expand.
+type FileRequestOptions struct {
+	Timeout uint
```

Comment:
+1 on adding documentation. Not on using `time.Duration` because all other funcs use `int` or `uint`.